From bc4fa495318d5addf15d5ce4c48d6997780f7a1c Mon Sep 17 00:00:00 2001
From: Zhong Hongbo <hongbo.zhong@windriver.com>
Date: Fri, 10 Oct 2014 10:15:24 +0800
Subject: [PATCH 1810/1825] crypto: convert out/in_be32 to iowrite/ioread32be
 function

The out/in_be32 just only be defined by the ppc platform.

Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 crypto/ocf/talitos/talitos.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/crypto/ocf/talitos/talitos.c b/crypto/ocf/talitos/talitos.c
index 185d2b6..516f1a5 100644
--- a/crypto/ocf/talitos/talitos.c
+++ b/crypto/ocf/talitos/talitos.c
@@ -178,6 +178,11 @@ int talitos_debug = 0;
 module_param(talitos_debug, int, 0644);
 MODULE_PARM_DESC(talitos_debug, "Enable debug");
 
+#ifdef CONFIG_ARM
+#define in_be32(addr)		ioread32be(addr)
+#define out_be32(addr, val)	iowrite32be(val, addr)
+#endif
+
 static inline void talitos_write(volatile unsigned *addr, u32 val)
 {
         out_be32(addr, val);
-- 
1.7.5.4

